<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Autoritation</class>
 <widget class="QWidget" name="Autoritation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="signInButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>85</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Вход</string>
   </property>
  </widget>
  <widget class="QLabel" name="topLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Вход</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginLine">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLine">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Неверный логин или пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="errPic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>155</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
